admin/database/database.go

```diff
@@ -313,6 +315,7 @@ type UpdateOrganizationOptions struct {
 	QuotaStorageLimitBytesPerDeployment int64
 	BillingCustomerID                   string
 	PaymentCustomerID                   string
+	BillingEmail                        string
```

Comment:
This is missing update in a few places.

Reply:
I fixed them here - https://github.com/rilldata/rill/pull/5312/commits/de29f94cca5628533311fb0541f9fb9ed3fbb0ce

Reply:
Sorry missed latest changes :)